src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/NuGetProjectService.cs

```diff
@@ -78,23 +78,24 @@ public async Task<InstalledPackagesResult> GetInstalledPackagesAsync(Guid projec
 
         private async Task<(InstalledPackageResultStatus, IReadOnlyCollection<NuGetInstalledPackage>)> GetInstalledPackagesAsync(BuildIntegratedNuGetProject project, CancellationToken cancellationToken)
         {
-            NuGetInstalledPackage ToNuGetInstalledPackage(PackageReference packageReference, FallbackPackagePathResolver pathResolver)
+            NuGetInstalledPackage ToNuGetInstalledPackage(PackageReference packageReference, FallbackPackagePathResolver pathResolver, bool directDependency)
             {
                 var id = packageReference.PackageIdentity.Id;
 
-                var versionRange = packageReference.AllowedVersions;
                 string requestedRange =
-                    packageReference.AllowedVersions.OriginalString // most packages
-                    ?? packageReference.AllowedVersions.ToShortString();
-                string version = versionRange.MinVersion.OriginalVersion ?? versionRange.MinVersion.ToNormalizedString();
-                var installPath = pathResolver.GetPackageDirectory(id, version);
-                bool directDependency = true;
+                    packageReference.AllowedVersions?.OriginalString // most packages
+                    ?? packageReference.AllowedVersions?.ToShortString();
```

Comment:
If `OriginalString` is null, would it make sense to call `ToShortString()` ? on null OriginalString data 

```suggestion
                    packageReference.HasAllowedVersions?
                    (  packageReference.AllowedVersions.ToShortString()
                    ?? packageReference.AllowedVersions.OriginalString) // most packages
                    : null;
```

Reply:
>If OriginalString is null, would it make sense to call ToShortString() on null OriginalString data?

Umm, I don't understand what you're asking. We're not calling `ToShortString()` on `OriginalString`, we're calling it on the parent object.

Your suggestion flips the order of the two, but I prefer `OriginalString`, because it should be the string that's used in the `Version="..."` attribute in the csproj. allowing callers like Roslyn to theoretically install the same string into a different project, keeping all the projects in the solution consistent. Normalizing it via `ToShortString()` will give the functionally equivalent result, but if someone wrote the non-normalized version in their csproj, then it might have been for a reason.